public/static/locales/en/data.json

```diff
@@ -18,7 +18,7 @@
     "cancel": "Cancel",
     "communityData": "Community Data",
     "copyPath": "Copy Path",
-    "copyLinks": "Copy Public Link(s)",
+    "copyLinks": "Copy",
```

Comment:
I'd be tempted to leave this alone, but change the button label to use `common:copy` instead 🤔